homeassistant/components/tplink/strings.json

```
@@ -42,16 +42,36 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
+      },
+      "reconfigure": {
+        "title": "Reconfigure TPLink entry",
+        "description": "Will update configuration for device {mac}",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "camera_auth_confirm": {
+        "title": "Camera live view",
+        "description": "Input device camera account credentials. Leave blank if they are the same as your TPLink cloud credentials.",
+        "data": {
+          "live_view": "Enable camera live view",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
       }
     },
     "error": {
       "cannot_connect": "Connection error: {error}",
-      "invalid_auth": "Invalid authentication: {error}"
+      "invalid_auth": "Invalid authentication: {error}",
+      "invalid_camera_auth": "Authentication error from camera stream",
+      "cannot_connect_camera": "Cannot access camera stream, check camera account is enabled",
+      "both_or_none": "Username and password must both be blank or not blank"
```

```suggestion
      "invalid_auth": "Unable to authenticate: {error}",
      "invalid_camera_auth": "Camera stream authentication failed",
      "cannot_connect_camera": "Unable to access the camera stream, verify that you have set up the camera account",
      "both_or_none": "You have to set both username and password, or leave them both empty"
```
Unsure if all of these should be complete sentence and end with a period, but here's my suggestions how to reword them a bit more end user friendly :-)